### PR TITLE
Add "ok-to-test" label to ISO and kicbase PRs

### PR DESCRIPTION
**Pull request by ilya-zuyev**
Our Jenkins ISO and kicbase release jobs create new PRs with updated versions when a build is complete. 
This PR changes release scripts to make those new ISO/kicbase PRs "ok-to-test" automatically and start integration tests

### CODE REVIEW

**Review by medyagh — APPROVED** [review]

File: hack/jenkins/build_iso.sh [hunk 1]
```diff
@@ -101,5 +101,8 @@ else
 	git remote add minikube-bot git@github.com:minikube-bot/minikube.git
 	git push -f minikube-bot ${branch}
 
-	gh pr create --fill --base master --head minikube-bot:${branch}
+	pr=$(gh pr create --fill --base master --head minikube-bot:${branch} \
+	      | grep -o -E "github.com/kubernetes/minikube/pull/[0-9]+" \
```
Comment (medyagh) [inline hack/jenkins/build_iso.sh:105]: what is the extra unrelated changes meant for ?

Comment (ilya-zuyev) [reply hack/jenkins/build_iso.sh:105]: what do you mean? we get PR number on lines 104-105 parsing `gh pr create` output and then adding `/ok-to-test` comment on line 107. All changes are important here.

File: hack/jenkins/kicbase_auto_build.sh [hunk 2]
```diff
@@ -126,5 +126,8 @@ else
 	git remote add minikube-bot git@github.com:minikube-bot/minikube.git
 	git push -f minikube-bot ${branch}
 
-	gh pr create --fill --base master --head minikube-bot:${branch}
+	pr=$(gh pr create --fill --base master --head minikube-bot:${branch} \
+	      | grep -o -E "github.com/kubernetes/minikube/pull/[0-9]+" \
+	      | grep -o -E '[0-9]+')
+	gh pr comment "$pr" --body "/ok-to-test"
```
Comment (medyagh) [inline hack/jenkins/kicbase_auto_build.sh:132]: is there a way we could add the /ok-to-test only if the label is not already set ?

I think if the label is already set, the Image Push commit will start a job, and this comment will start another one, and it will make two jobs running per commit wdyt ?

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/jenkins/kicbase_auto_build.sh (hunk 2 above)
Comment (sharifelgamal) [reply hack/jenkins/kicbase_auto_build.sh:132]: gh doesn't have a great of checking labels on a specific PR unfortunately

Comment (ilya-zuyev) [reply hack/jenkins/kicbase_auto_build.sh:132]: we just created this PR, there is no way `/ok-to-test` is set there :)
